game_template/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
 
   audioplayers: ^1.1.0
   cupertino_icons: ^1.0.2
-  go_router: ^5.0.5
+  go_router: ^5.2.1
```

Comment:
Raising dependency version lower bound because I'm not 100% sure that the new code works as expected with the previous versions.